projects/storefrontlib/cms-structure/guards/cms-page.guard.ts

```diff
@@ -28,11 +29,14 @@ export class CmsPageGuard {
   constructor(
     protected routingService: RoutingService,
     protected cmsService: CmsService,
+    /** since 2211.24 not used anymore, but called indirectly via {@link BeforeCmsPageGuardService} */
```

Comment:
`query:` Do we have information if customers extend this guard? If yes, maybe it's worth deprecating the whole constructor and adding new one without `ProtectedRoutesGuard`.